### PR TITLE
Model Testing

### LINKED ISSUE #69: Make more tests for the users and crypto models
As required by the last submission, all the methods for our current data models should be tested.

Time Estimate: 2 Hours

### PR DESCRIPTION
**Pull request by WeslyOrtega**
Created extensive testing to both the User model and Crypto model.
Also reformatted the previous tests.

Closes #69 

Time Spent: 2 Hours

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,5 +13,3 @@ install:
 # command to run tests
 script:
   - pytest -v backend/test_*.py
-  # - coverage run -m  pytest -v
```
Comment (jinyung2) [inline .travis.yml:16]: what did the 'coverage' do in previous iterations?

**Review by jinyung2 — APPROVED** [review]
Approved and looks good, but I did have some questions regarding pytest. Does it provide some kind of mocking of functions and/or db models? It seems like the current tests access the actual DB and I was wondering if it would be appropriate to have a separate document in mongoDB for testing or just mock the database access entirely?

File: .travis.yml (hunk 1 above)
Comment (WeslyOrtega) [reply .travis.yml:16]: Nothing. They were in the professor's sample file, but we couldn't get them to work, so we had commented them out.

**Comment by jinyung2** [comment]
I think extensions on testing are unnecessary at this point in the project but if we wanted to continue working on the app in the future that would be a possibility. New changes look good!

**Pull request merged**